tests/utils.py

```diff
@@ -17,3 +17,17 @@ def load_fixture_file(filename):
         filename
     )
     return open(path).read()
+
+
+def error_code_in_result(error_code, result):
+    """
+    assert that a specific error_code is into the results
+
+    :param error_code:
+    :param result:
+    :return:
+    """
+    for element in result:
+        if error_code in element[2]:
+            return True
+    return False
```

Comment:
@avallbona could you add a new line at the end of the file? Thanks!

```suggestion
    return False

```

Reply:
done